### PR TITLE
Added electric cash coin

### PR DESCRIPTION
**Pull request by kj-crypto**

### CODE REVIEW

**Review by PJLewandowski — COMMENTED** [review]
Shouldnt we also update requirements for deterministic build?

File: electrum/constants.py [hunk 1]
```diff
@@ -152,17 +152,53 @@ class BitcoinSimnet(BitcoinTestnet):
     LN_DNS_SEEDS = []
 
 
+class ElectricCashMainnet(BitcoinMainnet):
+
+    WIF_PREFIX = 0x80
+    ADDRTYPE_P2PKH = 33
+    ADDRTYPE_P2SH = 87
+    SEGWIT_HRP = "elcash"
+    GENESIS = "0000000028ce26975b32feda3d75ac3fe10372f75062366cfba4e934dcc6a48b"
```
Comment (PJLewandowski) [inline electrum/constants.py:161]: This is BTCV mainnet genesis block hash.

File: electrum/constants.py [hunk 2]
```diff
@@ -152,17 +152,53 @@ class BitcoinSimnet(BitcoinTestnet):
...
+    DEFAULT_SERVERS = read_json('servers_elcash.json', {})
```
Comment (PJLewandowski) [inline electrum/constants.py:162]: I think we could add these new config files, even if they will be empty for now.

File: electrum/constants.py (hunk 1 above)
Comment (kj-crypto) [reply electrum/constants.py]: fixed

File: electrum/constants.py (hunk 2 above)
Comment (kj-crypto) [reply electrum/constants.py:162]: done

**Review by PJLewandowski — APPROVED** [review]

**Pull request merged**